bin/gftools-build-vf.py

```diff
@@ -180,77 +233,214 @@ def get_style_list():
     printG("    [!] Done")
 
 
-def run_fontmake():
+def run_fontmake_variable():
     """
-    Builds ttf fonts files with font make.
+    Builds ttf variable font files with FontMake.
     """
     for source in sources:
-        print("\n**** Building %s font files with Fontmake:" % source)
+        print("\n**** Building %s variable font files with FontMake:" % source)
+        print("     [+] Run: fontmake ")
+        if args.ufosrc == True:
+            subprocess.call(
+                "fontmake \
+                          -g source/master_ufo/%s.designspace \
+                          -o variable \
+                          --output-path fonts/%s-VF.ttf"
+                % (source, source),
+                shell=True,
+            )
+        else:
+            subprocess.call(
+                "fontmake \
+                          -g source/%s.glyphs \
+                          -o variable \
+                          --output-path fonts/%s-VF.ttf"
+                % (source, source),
+                shell=True,
+            )
+        print("     [!] Done")
+    printG("    [!] Done")
+
+
+def run_fontmake_static():
+    """
+    Builds ttf static font files with FontMake.
+    """
+    for source in sources:
+        print("\n**** Building %s static font files with FontMake:" % source)
         print("     [+] Run: fontmake ")
         subprocess.call(
             "fontmake \
-                      -g sources/%s.glyphs \
-                      -o variable \
-                      --output-path fonts/%s-VF.ttf \
-            > /dev/null 2>&1"
-            % (source, source),
+                      -g source/%s.glyphs \
+                      -o ttf \
+                      --keep-overlaps -i"
```

Comment:
Why keep overlaps for static fonts?

Reply:
Oh, thanks! That's a mistake and shouldn't be there. I'll remove it.

This script needs some work and I think most other people I have talked to think just using BASH scripts is the better way to go. The idea for this tool was to make a very user friendly VF build script for people who use Python but might not be familiar with shell scripting.